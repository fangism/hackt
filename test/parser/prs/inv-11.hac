// "inv-11.hac" stupid, and rejected

defproc inv(bool a, b) {
prs {
	~~a -> b-	// not-not!
	~a -> b+
}
}

