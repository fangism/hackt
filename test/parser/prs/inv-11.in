// "inv-11.in" stupid, and rejected

defproc inv(bool a, b) {
prs {
	~~a -> b-	// not-not!
	~a -> b+
}
}

