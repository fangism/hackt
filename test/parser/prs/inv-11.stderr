parse error: syntax error
parser stacks:
state	value
0	(null) 
34	keyword: defproc [3:1..7]
112	identifier: inv [3:9..11]
181	list<(port-formal-decl)>: (port-formal-decl) ... [3:12..22]
254	{ [3:24]
292	keyword: prs [4:1..3]
214	{ [4:5]
271	~ [5:2]
333	~ [5:3]
in state 333, possible rules are:
	prs_not : '~' . prs_unary_expr  (197)
acceptable tokens are: 
	'(' (shift)
	ID (shift)
	SCOPE (shift)
